docs/compatibility.md

```diff
@@ -112,3 +112,14 @@ Before [PR 4939](https://github.com/open-mmlab/mmdetection/pull/4939), since `py
 If MMDetection is installed before Detectron2, they could work under the same environment.
 [PR 4939](https://github.com/open-mmlab/mmdetection/pull/4939) deprecates mmpycocotools in favor of official pycocotools.
 Users may install MMDetection and Detectron2 under the same environment after [PR 4939](https://github.com/open-mmlab/mmdetection/pull/4939), no matter what the installation order is.
+
+## SSD compatibility
```

Comment:
Should put it on the top as it is the nearest.
We can make it a subsection of MMDetection v2.14.0.